e2e/panels-suite/frontend-sandbox-panel.spec.ts

```diff
@@ -64,15 +61,10 @@ describe('Panel sandbox', () => {
 
   describe('Sandbox enabled', () => {
     beforeEach(() => {
-      e2e.flows.openDashboard({
-        uid: DASHBOARD_ID,
-        queryParams: {
-          '__feature.pluginsFrontendSandbox': true,
-        },
-      });
       cy.window().then((win) => {
         win.localStorage.setItem('grafana.featureToggles', 'pluginsFrontendSandbox=1');
       });
+      cy.reload();
```

Comment:
Is this necessary?

Reply:
Yes! otherwise the frontend won't read the localstorage flag.